### PR TITLE
translation(cy): translate cartpole and gym-hybrid env tutorial into English

### PR DESCRIPTION
**Pull request by chenyingvivian**

### CODE REVIEW

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: Gym_en.rst [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+=========
+
+action space
+------------
+
+The action space of Gym-hybrid belongs to the mixed space of discrete continuous actions. There are 3 discrete actions: Accelerate, Turn, and Break. The actions Accelerate and Turn need to give the corresponding 1-dimensional continuous parameters.
+
+- \ ``Accelerate (Acceleration value)`` \: Indicates that the agent is accelerated by the size of \ ``acceleration value`` \. The value range of \ ``Acceleration value`` \ is \ ``[0,1]`` \. The numeric type is \``float32``.
+  
+- \ ``Turn (Rotation value)`` \: means to turn the agent in the direction of \ ``rotation value`` \. The value range of \ ``Rotation value`` \ is \ ``[-1,1]`` \. The numeric type is \``float32``.
```
Comment (VaninaY) [inline Gym_en.rst:51]: - \ ``Turn (Rotation value)`` \: means to turn the agent in the direction of \ ``rotation value`` \. The value range of \ ``Rotation value`` \ is \ ``[-1,1]`` \. The numeric type is \ ``float32``.

File: Gym_en.rst [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+  
+- \ ``Break ()`` \: means stop.
+
+Using the gym environment space definition can be expressed as:
+
+.. code::python
```
Comment (VaninaY) [inline Gym_en.rst:57]: .. code:: python

File: Gym_en.rst [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    action_space = spaces.Tuple((spaces.Discrete(3),
+                                    spaces.Box(low=0, high=1, shape=(1,)),
+                                    spaces.Box(low=-1, high=1, shape=(1,))))
+
+state space
+------------
+
+The state space of Gym-hybrid is represented by a list of 10 elements, which describes the current state of the agent, including the agent's current coordinates, velocity, the sine and cosine of the orientation angle, the coordinates of the target, the distance between the agent and the target, and the target. Distance-related bool value, current relative steps.
+
+.. code::python
```
Comment (VaninaY) [inline Gym_en.rst:70]: .. code:: python

File: Gym_en.rst [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+                0 if distance > target_radius else 1,
+                current_step / max_step
+            ]
+
+Bonus space
+-------------
+The reward of each step is set as the length of the agent's distance from the target after the action is performed in the previous step minus the length of the distance from the target after the current step performs the action, ie \``dist_t-1 - dist_t`` \. The algorithm has a built-in \``penalty``\ to motivate the agent to be faster
+Achieve goals. When the episode ends, if the agent stops in the target area, it will get an additional reward with a value of 1; if the agent goes out of bounds or exceeds the maximum number of steps of the episode, it will not get additional reward. The reward at the current moment is expressed by the formula as follows:
+
+.. code::python
```
Comment (VaninaY) [inline Gym_en.rst:90]: .. code:: python

File: Gym_en.rst [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+- The agent successfully entered the target area
+  
+- agant out of bounds
+  
+- Reach the maximum step of the episode
+  
+
+built-in environment
+------------
+There are two built-in environments, \``"Moving-v0"``\ and \``"Sliding-v0"``\. The former does not consider conservation of inertia, while the latter does (so is more practical). The two environments are consistent in state space, action space, and reward space.
```
Comment (VaninaY) [inline Gym_en.rst:108]: There are two built-in environments,\ ``"Moving-v0"`` \and \ ``"Sliding-v0"`` \. The former does not consider conservation of inertia, while the latter does (so is more practical). The two environments are consistent in state space, action space, and reward space.

File: Gym_en.rst [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+
+Store video
+--------
+
+Some environments have their own rendering plug-ins, but DI-engine does not support the rendering plug-ins that come with the environment, but generates video recordings by saving the logs during training. For details, please refer to the Visualization & Logging section under the DI-engine `official documentation <https://opendilab.github.io/DI-engine/quick_start/index.html>`__ Quick start chapter.
+
+DI-zoo runnable code example
+======================
+
+The following provides a complete gym hybrid environment config, using DDPG as the baseline algorithm. Please run the \``gym_hybrid_ddpg_config.py``\ file in the \``DI-engine/dizoo/gym_hybrid``\ directory, as follows.
```
Comment (VaninaY) [inline Gym_en.rst:121]: The following provides a complete gym hybrid environment config, using DDPG as the baseline algorithm. Please run the \ ``gym_hybrid_ddpg_config.py`` \ file in the \ ``DI-engine/dizoo/gym_hybrid`` \ directory, as follows.

File: Gym_en.rst [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+
+.. code::python
```
Comment (VaninaY) [inline Gym_en.rst:123]: .. code:: python

File: cartpole_en.rst [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+.. image:: ./images/cartpole.gif
+   :align: center
+
+Install
+====
+
+installation method
+--------
+
+The Cartpole environment is built into the gym, and you can install the gym directly. Its environment id is \``CartPole-v0``\.
```
Comment (VaninaY) [inline cartpole_en.rst:17]: The Cartpole environment is built into the gym, and you can install the gym directly. Its environment id is \ ``CartPole-v0`` \.

File: cartpole_en.rst [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+
+.. code:: shell
```
Comment (VaninaY) [inline cartpole_en.rst:19]: .. code:: shell

File: cartpole_en.rst [hunk 10]
```diff
@@ -0,0 +1,218 @@
...
+
+The action space of Cartpole belongs to the discrete action space, and there are two discrete actions, namely left shift and right shift.
+
+- \ ``Left Move`` \: 0 means to move the agent to the left.
+
+- \ ``Right move`` \: 1 means to move the agent to the right.
+
+Using the gym environment space definition can be expressed as:
+
+.. code::python
```
Comment (VaninaY) [inline cartpole_en.rst:49]: .. code:: python

File: cartpole_en.rst [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+State space
+------------
+
+Cartpole's state space has 4 elements, which are:
+
+- \ ``Cart Position`` \: Cart position, in the range \ ``[-4.8, 4.8]`` \.
+  
+- \ ``Cart Velocity`` \: The speed of the cart, in the range \ ``[-inf, inf]`` \.
+
+- \``Pole Angle``\: The angle of the pole, in the range \``[-24 deg, 24 deg]``\.
```
Comment (VaninaY) [inline cartpole_en.rst:62]: - \ ``Pole Angle`` \: The angle of the pole, in the range \ ``[-24 deg, 24 deg]`` \.

File: cartpole_en.rst [hunk 12]
```diff
@@ -0,0 +1,218 @@
...
+
+- \``Pole Angular Velocity``\: The angular velocity of the pole, in the range \``[-inf, inf]``\.
```
Comment (VaninaY) [inline cartpole_en.rst:64]: - \ ``Pole Angular Velocity`` \: The angular velocity of the pole, in the range \ ``[-inf, inf]`` \.

File: cartpole_en.rst [hunk 13]
```diff
@@ -0,0 +1,218 @@
...
+--------
+
+Some environments have their own rendering plug-ins, but DI-engine does not support the rendering plug-ins that come with the environment, but generates video recordings by saving the logs during training. For details, please refer to the Visualization & Logging section under the DI-engine `official documentation <https://opendilab.github.io/DI-engine/quick_start/index.html>`__ Quick start chapter.
+
+DI-zoo runnable code example
+======================
+
+The following provides a complete cartpole environment config, using the DQN algorithm as the policy. Please run the \ ``cartpole_dqn_main.py`` \ file in the \ ``DI-engine/dizoo/classic_control/cartpole/entry`` \ directory, as follows.
+
+.. code::python
```
Comment (VaninaY) [inline cartpole_en.rst:102]: .. code:: python

File: Gym_en.rst [hunk 14]
```diff
@@ -0,0 +1,213 @@
+gym_hybrid_en.rst
```
Comment (PaParaZz1) [inline Gym_en.rst:1]: rename this file to `gym_hybrid_en.rst`, and rename the head to `gym hybrid`

File: Gym_en.rst [hunk 15]
```diff
@@ -0,0 +1,213 @@
...
+.. code:: python
+
+    import gym
+    import gym_hybrid
+    env = gym.make('Moving-v0')
+    obs = env.reset()
+    print(obs)
+
+Environment Introduction
+=========
```
Comment (PaParaZz1) [inline Gym_en.rst:42]: it should be longer than the head

File: Gym_en.rst [hunk 16]
```diff
@@ -0,0 +1,213 @@
+gym_hybrid_en.rst
+~~~~~~~~~~~~~~~~~~~
```
Comment (PaParaZz1) [inline Gym_en.rst:2]: move these two files to `source/env_tutorial`

File: Gym_en.rst [hunk 17]
```diff
@@ -0,0 +1,213 @@
...
+    gym_hybrid_ddpg_create_config = EasyDict(gym_hybrid_ddpg_create_config)
+    create_config = gym_hybrid_ddpg_create_config
+
+
+    if __name__ == "__main__":
+        serial_pipeline([main_config, create_config], seed=0)
+
+References
+======================
+- Gym-hybrid `source code <https://github.com/thomashirtz/gym-hybrid>`__
```
Comment (PaParaZz1) [inline Gym_en.rst:200]: No need for capitalization here

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

**Comment by PaParaZz1** [comment]
This PR has been integrated in #135

**Pull request closed without merging**